st2client/st2client/commands/resource.py

```diff
@@ -731,20 +732,47 @@ def __init__(self, resource, *args, **kwargs):
 
         self.parser.add_argument(argument, metavar=metavar, help=help)
 
+        self.parser.add_argument(
+            "--yes",
```

Comment:
Can you add an option for `-y`?

Reply:
> 
> 
> Can you add an option for `-y`?

`-y` argument is already used for yaml formatting at https://github.com/StackStorm/st2/blob/master/st2client/st2client/commands/resource.py#L163. So it is conflicting. If any other suitable option is there as an argument, I will add it.

Reply:
How about `-f` or `--force` to be consistent with `rm` command?

Reply:
Can we change to -f and --force?